src/db/migrations/28-add-mighty-33-others-die.ts

```diff
@@ -0,0 +1,54 @@
+import { Knex } from 'knex';
+
+import { IdExtractorTypes, TitleExtractorTypes } from '../../types/fieldExtractor';
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatform, MusicPlatformType } from '../../types/platform';
+import { ethereumArtistId } from '../../utils/identifiers';
+import { addNftFactory, addPlatform, removeNftFactory, removePlatform } from '../migration-helpers';
+
+
+const mighty33Address = '0x8C62dD796e13aD389aD0bfDA44BB231D317Ef6C6';
+
+const MIGHTY_33: MusicPlatform = {
+  id: mighty33Address,
+  type: MusicPlatformType['multi-track-multiprint-contract'],
+  name: 'Mighty33',
+}
```

Comment:
it seems like the platform is actualy https://geniuscorp.fr/ ? let's maybe keep it as that
```suggestion
const MIGHTY_33: MusicPlatform = {
  id: 'geniuscorp.fr',
  type: MusicPlatformType['multi-track-multiprint-contract'],
  name: 'geniuscorp.fr',
}
```